### ISSUE TITLE
Error while downloading...

### ISSUE THREAD

**Issue by c0ppo** [riazXrazor/udemy-dl#31]
I've been using this wonderful script for quite some time now. And everything worked like a charm.
But when I tried to download today, I got some errors. I've tried updating npm, udl, reinstalling everything, but nothing seems to help.

I'm on MacOS, latest version (non beta). This is the error that I get:

```
/usr/local/lib/node_modules/udemy-dl/lib/core.js:53
      cb(r,b.match(/name='csrfmiddlewaretoken'\s+value='(.*)'/)[1]);
                                                               ^

TypeError: Cannot read property '1' of null
    at Request._callback (/usr/local/lib/node_modules/udemy-dl/lib/core.js:53:64)
    at Request.self.callback (/usr/local/lib/node_modules/udemy-dl/node_modules/request/request.js:186:22)
    at Request.emit (events.js:180:13)
    at Request.<anonymous> (/usr/local/lib/node_modules/udemy-dl/node_modules/request/request.js:1163:10)
    at Request.emit (events.js:180:13)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/udemy-dl/node_modules/request/request.js:1085:12)
    at Object.onceWrapper (events.js:272:13)
    at IncomingMessage.emit (events.js:185:15)
    at endReadableNT (_stream_readable.js:1101:12)
    at process._tickCallback (internal/process/next_tick.js:114:19)
```

**Comment by riazXrazor** [comment]
May be udemy site ws down at that time, coz i jst tried the script its
working.

Plz try  to login again.

**Comment by c0ppo** [comment]
Nope, Udemy was online. And is still. And still not working for me.
But it's not the scripts fault, since it does work on my laptop. I'm gonna try to fix it for my computer, but since laptop is working like expected, I'm gonna use that for now.

Thank you for the really quick response.

**Comment by riazXrazor** [comment]
Aaa i see, btw let me kno if u find anything ok. :+1:

**Comment by davidhenley** [comment]
Same issue. Why is this closed?

**Comment by c0ppo** [comment]
I've closed it, because I've thought it was only me, a local problem with one computer.
But it appears that this wasn't the case. Now my laptop is stuck as well.

But I did this a few hours ago, on my desktop and my laptop:

`npm install -g udemy-dl
`

Without deleting, I've just run install script again. Restarted terminal, and it works. At least for now. I haven't been able to find what causes the problem in the first place, but this helped. At least for now.
